### DISCUSSION TITLE
[Feature Request] CMS Support, perhaps DecapCMS as a start?

### DISCUSSION THREAD

**Discussion by aaronlippold** [vuepress-theme-hope/vuepress-theme-hope#4512] [Ideas]
### Checklist

- [X] I searched existing issues and no one else requests similar feature.
- [X] I think that 25%+ users are positive towards this feature.

### Describe the feature

Given that many users use Hope as a documentation template, would possible integration to a git based CMS be something to consider?

**Comment by aaronlippold** [comment]
As an example, I use it to support all my security automation classes that I teach - https://mitre.github.io/saf-training/ - and being able to put all my classes into CMS would be very helpful.

**Comment by Mister-Hope** [comment]
Except markdown-it is sync so the markdown part is sync, all other parts of VuePress are async.


https://vuejs.press/advanced/cookbook/adding-extra-pages.html

You can add pages from cms with your own script easily, which should be similar with VitePress.

**Comment by Mister-Hope** [comment]
Besides, if you are interested in creating a similar plugin that can support a set of CMS, we can work together to build one at vuepress/ecosystem repo.

I can write most thing of that plugin, except plugin options, the content fetching part from cms and the related docs
